### DISCUSSION TITLE
Vertical Display of folders

### DISCUSSION THREAD

**Discussion by BayGallivanter** [RealRaven2000/QuickFolders#568] [Announcements]
I struggle to use Thunderbird efficiently - I have 12 email accounts most with sub folders!  I have come to realise that the folders are basically bookmarks to the folders in the mail client with most of the email management functionality of the T/B client.

Could these Quick Folders be displayed as a vertical bar to the left of the T/b folder system?  You did mention in one of your help section about "looking up there" which in my mind is not a natural way to work. With a vertical display of folders it is more natural as we tend to move our our left to right (well in the western worlds we do).

**Comment by RealRaven2000** [comment]
Thanks for the suggestion!

I have seen the idea before, but this is quite hard to achieve

1 - QuickFolders is implemented as a xul toolbar element - which are currently always horizontal, so it would be not so easy to make a vertical version

2 - subfolder menus. One of QuickFolders main functions is managing email for users who have complex folder tree configurations (more than 100 folders). Usually we would find a dropdown with many subfolders that can serve as jump or drag targets:

![PkcatNujcgfA0tWP](https://github.com/user-attachments/assets/64fe5db3-1cb0-44e4-854b-61986c6298e9)

I find that using the mouse, dragging horizontally instead of vertically for moving mail is physically harder (in my experience controlled sideways movements on usually large wide screens are harder  with the mouse because it strains the hand joint, whereas I can use fingers for vertical movement, especially when using a wrist rest) and the menu would become awkward to use as it will sometime necessitate dragging right-up or right-down to find a target, depending on the vertical position of the QF tab.

![CtgaHM0bJSgMcUJ0](https://github.com/user-attachments/assets/098697e0-ca2f-467e-a992-84da24991c70)

To me, this movement is cramped and highly RSI-inducing. I decided to completely avoid using the tree for all folder viewing and mail management, and only use QuickFolders tabs and the quickMove field in order to access all other folders:

![FLAOSdY7hprqo2pl](https://github.com/user-attachments/assets/426a8ef1-d943-43bb-bd4c-a174ccb251c5)
